### PR TITLE
Create ROADMAP.md

### LINKED ISSUE #106: Create a ROADMAP
Roadmap for the style guides.

### PR DESCRIPTION
**Pull request by arfon**
Initial stab at a roadmap for this project. Some inline comments below...

/ cc @spacetelescope/style-gurus 

Fixes #106

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- Security guide
+- Guidelines for how to advertise compliance with STScI Style Guides (https://github.com/spacetelescope/style-guides/issues/102)
+
+### v1.1 release (release date June 2020)
+
+- Merge/reconcile [INS standards for JWST community software](https://github.com/spacetelescope/ins-jwst-community-software) with STScI Style Guides.
+- Web pages guide
+- Extended security guide
+
+## Other key activities
```
Comment (arfon) [inline ROADMAP.md]: Some other things we might want to address:

- Figure out who the actual maintainers are of this.
- How we decide when to add a new guide.

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- Review of style guides with Engineering Steering Committee.
+- Seek endorsement of these guides from major missions (HST, JWST, WFIRST).
```
Comment (arfon) [inline ROADMAP.md]: After endorsement, we might want to ask the missions to encourage/mandate adoption by key teams across the Institute?

It sounds like JWST are already doing this to some extent in INS.

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- GitHub repositories guide
+- Jupyter Notebooks guide
+- Python (naming, coding standards, testing etc.) guide
+- Python packaging guide
+- Security guide
+- Guidelines for how to advertise compliance with STScI Style Guides (https://github.com/spacetelescope/style-guides/issues/102)
+
+### v1.1 release (release date June 2020)
+
+- Merge/reconcile [INS standards for JWST community software](https://github.com/spacetelescope/ins-jwst-community-software) with STScI Style Guides.
```
Comment (arfon) [inline ROADMAP.md]: / cc @bourque for 👀

File: ROADMAP.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Style Guides Roadmap
+
+> tl;dr The goal of this document is to document the priority items for style guide development over the next 3-6 months as well as key activities designed to grow awareness and adoption of the Style Guides at STScI.
```
Comment (pllim) [inline ROADMAP.md]: 3-6 months from *when*? 😉  Given the current development rate, perhaps 3-6 mo range is too aggressive?

In addition, should we also include some longer term items here?

File: ROADMAP.md (hunk 1 above)
Comment (pllim) [reply ROADMAP.md]: Getting that mission support (mentioned below) would be essential in recruiting and retaining maintainers, but that is a separate conversation.

File: ROADMAP.md (hunk 2 above)
Comment (pllim) [reply ROADMAP.md]: Yes, on the JWST side, I think @bourque and @shanosborne are the main developers for style guides currently. (Please correct me if I am wrong.)

File: ROADMAP.md (hunk 4 above)
Comment (arfon) [reply ROADMAP.md]: > 3-6 months from when? 😉 Given the current development rate, perhaps 3-6 mo range is too aggressive?

😁January 2020 --> July 2020? Although perhaps we should say this is work we want to get done in calendar year 2020?

Comment (pllim) [reply ROADMAP.md]: All I am saying is that we should be realistic. 😉

File: ROADMAP.md (hunk 3 above)
Comment (bourque) [reply ROADMAP.md]: This date *should* be fine.  Within the next two months, @shanosborne and I will finish documenting all of the standards for the INS JWST community software tools, and will be meeting with the JWST mission office to make sure we are all on the same page moving forward.  So this is to say -- after that meeting, we should know how feasible a June 2020 release will be.

**Comment by pllim** [comment]
Didn't happen.

**Pull request closed without merging**